libgraph/include/katana/PropertyGraph.h

```diff
@@ -342,6 +342,12 @@ class KATANA_EXPORT PropertyGraph {
     return edge_entity_type_manager_;
   }
 
+  Result<RDKLSHIndex> LoadRDKLSHIndex() { return rdg_.LoadRDKLSHIndex(); }
```

Comment:
So these are the calls I will use as a user to initiate saving and loading?

Reply:
yep.